build.gradle.kts

```diff
@@ -1,11 +1,15 @@
 plugins {
-    id("com.lovelysystems.gradle") version ("1.6.2")
+    id("com.lovelysystems.gradle") version ("1.11.5")
 }
 
 lovely {
     gitProject()
-    dockerProject("lovelysystems/docker-postgres", stages = listOf("", "backup", "client", "upgrade12"))
-    with(dockerFiles) {
+    dockerProject(
+        "lovelysystems/docker-postgres",
+        stages = listOf("", "backup", "client", "upgrade12"),
+        platforms = listOf("linux/amd64"),
+        buildPlatforms = listOf("linux/amd64")
```

Comment:
my M1 mac was producing arm64 images which caused me a lot of troubles during testing, so I updated the gradle plugin version here and nailed the platform to amd64. As I've seen on Docker Hub, we were anyway uploading amd64 versions only, so I guess it's not a problem, but raise concerns if any.

Reply:
add a note regarding apgdiff copy from amd64